00-HTML-CSS-basics/ex1/1_1_6/index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Sing up</title>
+    </head>
+   
+    <body>
+        <form align="center">
+            <tr>
```

Comment:
Is really needed to use a `table` here? Is not a good practice to organize a from with a table

Reply:
This is more wrong than before. Remove all the table related stuff and work without it for forms